src/utils/network/session-core.js

```diff
@@ -163,7 +163,8 @@ export const sendMessageTTL = async (session, messageObject, podUrl) => {
   let recipientName;
 
   try {
-    recipientName = await getUserProfileName(session, recipientWebId);
+    const recipientUsername = recipientPodUrl.split('/')[2].split('.')[0];
```

Comment:
I actually didn't look at this part of the code too closely, but looking at it now, I find it kind of confusing. 

We take in a pod url where we will be sending the message to, which makes it kind of analogous to an email address. Then we attempt to find a username associated with that pod url. If we can't find a username, we error, saying we can't send the message.

Except we can send the message. We don't use the username to send, we use the pod url. Whether the username is correct or not is irrelevant to the pod url.

I feel like we can simplify this lines 163-170 to something like:
```
const recipientName = (await getUserProfileName(session, recipientWebId)) || "unknown user"
```

Reply:
Got this updated to using pod URLs as fallback values if profile name does not exist.

As for `message.title`, we previously had this as a required field to fill out so there wouldn't be a message with no title at this time. Think we adjust the validation for the `NewMessage` field to allow for empty titles in another PR.

Reply:
This looks good to me now